apps/todo-app/src/app/app.module.ts

```diff
@@ -23,7 +24,7 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
       }
     ),
     EffectsModule.forRoot([]),
-    !environment.production ? StoreDevtoolsModule.instrument() : []
+    !environment.production ? StoreDevtoolsModule.instrument() : [],
```

Comment:
I think you have the wrong configuration of eslint, because prettier always add a comma at end line